src/fsharp/CompileOps.fs

```diff
@@ -1774,20 +1774,25 @@ let OutputDiagnosticContext prefix fileLineFn os err =
 let GetFSharpCoreLibraryName () = "FSharp.Core"
 
 // If necessary assume a reference to the latest .NET Framework FSharp.Core with which those tools are built.
-let GetDefaultFSharpCoreReference() = typeof<list<int>>.Assembly.Location
+let GetDefaultFSharpCoreReference () = typeof<list<int>>.Assembly.Location
 
-// If necessary assume a reference to the latest System.ValueTuple with which those tools are built.
-let GetDefaultSystemValueTupleReference() = 
-#if COMPILER_SERVICE_AS_DLL
```

Comment:
@dsyme.
I enabled this reference assembly for FCS.  Let me know if you want this back under an #ifdef.

Reply:
It's ok I guess.  We should really add FCS documentation on the behaviour you get for default references